core/src/services/aliyun_drive/core.rs

```diff
@@ -78,6 +80,9 @@ impl Debug for AliyunDriveCore {
 
 impl AliyunDriveCore {
     async fn send(&self, mut req: Request<Buffer>, token: Option<&str>) -> Result<Buffer> {
+        // AliyunDrive raise NullPointerException if you haven't set a user-agent.
+        req.headers_mut()
+            .insert(header::USER_AGENT, HeaderValue::from_static("opendal"));
```

Comment:
Hi @Xuanwo, how can I get the OpenDAL version? I found that some of the AliyunDrive API requires a UA (otherwise they raise a NullPointerException 😅). Therefore, I want to add a UA in the format of `User-Agent: opendal/1.0.0`.

Reply:
Please use [opendal::raw::VERSION](https://docs.rs/opendal/latest/opendal/raw/constant.VERSION.html)